docs/docs/constraints.md

```diff
@@ -8,15 +8,15 @@ Constraints control where apps run to allow optimizing for either fault toleranc
 
 ## Field Names
 
-### Hostname
+### Hostname as field name
 
 Entering `@hostname` as the field name matches the agent node hostnames. See `UNIQUE operator`, below, for a usage example.
 
 All Marathon operators are supported when the field name is `@hostname`.
 
-### Region and zone
+### Region and zone as field names
 
-Use `@region` and `@zone` to configure [fault domain awareness and capacity extension]({{ site.baseurl }}/docs/fault-domain-awareness.html).
+Use the `@region` and `@zone` field names to configure [fault domain awareness and capacity extension]({{ site.baseurl }}/docs/fault-domain-awareness.html).
```

Comment:
Hmm, this seems off. (it was off before)

> Use the `@region` and `@zone` field names to configure fault domain awareness and capacity extension

A) The grammar is suffering. Should there be a "the" in there?
B) I don't think it's an extension. Support for Mesos fault domains is built-in to Marathon. It's on by default.
C) I just can't make sense of the sentence. The more I read it, the more confused I am.

Maybe this would be better?

> The `@region` and `@zone` fields refer to the respective [fault domain properties]({{ site.baseurl }}/docs/fault-domain-awareness.html). Specifying a constraint on `@region` will enable an instance to be placed in remote regions.